contents/handbook/strategy/roadmap.md

```diff
@@ -4,37 +4,43 @@ sidebar: Handbook
 showTitle: true
 ---
 
-Our mission is to increase the number of successful products in the world. 
+Our mission is to 
 
-Our roadmap for 2021 will do three things:
-1. Reach parity with other product analytics tools, but better
-2. Ensure we have the best developer platform for event-based analytics
-3. Make it easy to deploy and maintain a scalable version of PostHog, for free
+> "Increase the number of successful products in the world."
 
-# 1. Reach parity
+Our vision for 2023 is:
 
-PostHog went from 0 to a fully-featured analytics platform in an incredibly short period of time. Realistically, that did mean we had to cut some corners when it came to stability, usability and depth of analytics you could do with PostHog.
+> “Everyone building a product has a clear path to making it successful without losing control of their data.”
 
-In 2021, we're going to make sure that PostHog has all the features you love from other platforms. That way, self-hosting your product analytics doesn't need to be a compromise.
+Our roadmap for the second half of 2021 will do three things:
+1. Diagnosing causes
+2. Collaboration
+3. Easy to deploy, maintain and extend
 
-We're also putting more focus into usability and UX, especially for less technical users.
+# 1. Diagnosing Causes
 
-# 2. Best developer platform
+We've already shipped an incredibly powerful funnels product this year. However, as soon as you can see people dropping off in your funnel all you want to know is "why?". 
 
-Developers like using PostHog for many reasons. We're open-source at our core, which has helped us gain trust and adoption from the developer community.
-It's easy to debug, you can self-host and PostHog is now starting to become extensible.
+We aim to build a suite of tools that enable you two answer ths question of why through multiple methods:
+* Inidiviudal Qualitative: Observing session recordings and surveying users
+* Aggregated Qualitiative: Combining together hundreds of sessions to review behaviors of successufl and unsuccessful users at scale
+* Quantiative: Analysing patterns of successful and unsuccessful users to highlight clear opportunities to improve your product
 
-This year we're going to lean into that last item. We've kept plugins relatively quiet so far, but we believe plugins will be what will make PostHog the default choice for developers.
+# 2. Collaboration
+
+As larger teams and more complex organizations grow on PostHog, they need to work together to share insights and make decisions. Our focus on collaobration is key, not only to improve the quality and quantity of insights every user gets from PostHog, but also to ensure that users of PostHog get access to **only** the sensitive data which they **need** to access, ensuring user privacy.
```

Comment:
- Typo: "collaobration"
- Think collaboration is also important to ensure non-technical team members (or team members outside the scope of the Product Team) get access to all insights in a clear way.

Reply:
You can tell I'm lost without a spell checker... need to turn it on in VSCode